### DISCUSSION TITLE
How to build the final gene regulation network as the same cytoscape style (or other app) in the published paper.?

### DISCUSSION THREAD

**Discussion by JGangHan** [aertslab/scenicplus#204] [Q&A] [answered]
Hello,
Congulations for your online publication.
The gene regulation networks in that manuscript are very beauty and concise. 

But I can't build the same style GRN with example data or my own data by this cytoscape style file (https://github.com/aertslab/scenicplus/tree/main/cytoscape_styles)
Do you mind let me know how to generate the same style GRNs by cytoscape or other apps?
Kind regards,
Han.

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @JGangHan 

Thank you for the kind words.
I moved your question to the discussions section.

What do you mean by "same style GRN"? Are you talking about the shapes/colors/... of the nodes and edges or how the nodes and edges are organised?

For the former, the cytoscape file should solve this, let me know if your nodes have a different shape etc.

For the latter, there is no special sauce I'm afraid. We organised them by manually dragging them around in Cytoscape.

Best,

Seppe

**Reply by JGangHan** [reply]
Hi Seppe, 

Thanks for your response. I mean how to orgnize the nodes and edges as the GRNs in your manuscript (the latter one).
I will also do it manually.

Best wishes,
Han.

**Reply by SeppeDeWinter** [reply]
Good luck

Best,

Seppe
